src/inputs/ObjectSearch.tsx

```diff
@@ -233,7 +233,8 @@ class ObjectSearch extends Component<IObjectSearchProps> {
     // Select from search
     if (this.isMultiSelect) {
       const selectedOptionIds = selectedOption.map((_selectedOption: any) => _selectedOption.key)
-        , foundOptions = this.options.filter(option => selectedOptionIds.includes(option.id));
+        , optionsToSearch = Array.from(new Set([...this.injected.value, ...this.options]))
```

Comment:
Since we don't have an ORM, I don't _think_  `Array.from(new Set(` here will actually remove duplicates. It would keep both, as shown here:
```ts
Array.from(new Set([{ id: '1' }, { id: '1' }])) => [{ id: '1' }, { id: '1' }]
```
Perhaps instead we should do use [uniqBy](https://lodash.com/docs/4.17.15#uniqBy)?
```ts
_.uniqBy([{ id: '1' }, { id: '1' }], 'id').length => 1
```
like so:
```ts
        , optionsToSearch = uniqBy([...this.injected.value, ...this.options], 'id')
```